WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/LayoutPickerTracker.kt

```diff
@@ -3,6 +3,7 @@ package org.wordpress.android.ui.layoutpicker
 interface LayoutPickerTracker {
     fun trackPreviewModeChanged(mode: String)
 
+    // NOOP implementation provided to make override optional in subclasses
```

Comment:
Sneaking this comment in here to close the loop [here](https://github.com/wordpress-mobile/WordPress-Android/pull/16468#discussion_r864490457). :smile:

Reply:
Awesome, thank you for identifying this improvement and addressing it here 🙇